server/controllers/articleController.js

```diff
@@ -22,6 +22,11 @@ class ArticleController {
         let token = req.header('x-auth-token');
         const authorId = grabEmployeeIdFromToken(token, res);
         const currentDate = datetime.create().format('m/d/Y H:M:S');
+        // const articles = await this.articleModel().select('*', 'authorid=$1', [authorId]);
+        // const foundArticle = articles.find(a => a.title === title);
+        // if (foundArticle) {
+        //   return ResponseHandler.error(REQUEST_CONFLICT, 'The same article exists', res);
+        // }
```

Comment:
please remove commented lines of code @hakoemmy

Reply:
I am done